span.go

```diff
@@ -132,6 +133,12 @@ func setStringTag(s *ddSpan, key, v string) {
 		s.Resource = v
 	case ext.SpanType:
 		s.Type = v
+	case ext.AnalyticsEvent:
+		if v != "false" {
+			setMetric(s, ext.EventSampleRate, 1)
+		} else {
+			setMetric(s, ext.EventSampleRate, 0)
+		}
```

Comment:
Could/should the next line use `ext.SpanName` instead of `keySpanName`?

Reply:
It could! When I've added this there was no `ext.SpanName`